docs/devops-guide/secure-domain.md

```diff
@@ -1,132 +1,170 @@
 ---
 id: secure-domain
-title: Secure Domain setup
+title: Secure Domain Setup
 sidebar_label: Authentication (Secure Domain)
 ---
 
-It is possible to allow only authenticated users to create new conference
-rooms. Whenever a new room is about to be created, Jitsi Meet will prompt for
-a user name and password. After the room is created, others will be able to join
-from anonymous domain. Here's what has to be configured:
+It is possible to allow only authenticated users to create new conference rooms.
+Whenever a new room is about to be created, Jitsi Meet will prompt for a user
+name and password. After the room is created, others will be able to join from
+anonymous domain. Here's what has to be configured:
 
 ## Prosody configuration
 
-If you have installed Jitsi Meet from the Debian package, these changes should be made in `/etc/prosody/conf.avail/[your-hostname].cfg.lua`
+If you have installed Jitsi Meet from the Debian package, these changes should
+be made in `/etc/prosody/conf.avail/[your-hostname].cfg.lua`
+
+In the example below, this hostname is assumed to be `jitsi.example.com`. Update
+this value according to your own hostname.
 
 ### Enable authentication
 
-Inside the `VirtualHost "[your-hostname]"` block, replace anonymous authentication with hashed password authentication:
+Inside the `VirtualHost "[your-hostname]"` section, replace anonymous
+authentication with hashed password authentication:
 
 ```
-VirtualHost "jitsi-meet.example.com"
+VirtualHost "jitsi.example.com"
     authentication = "internal_hashed"
 ```
 
-Replace `jitsi-meet.example.com` with your hostname.
+You will see your own hostname instead of `jitsi.example.com` in your config
+file.
 
 ### Enable anonymous login for guests
 
-Add this block **after the previous VirtualHost** to enable the anonymous login method for guests:
+Add this section **after the previous VirtualHost** to enable the anonymous
+login method for guests:
 
 ```
-VirtualHost "guest.jitsi-meet.example.com"
-    authentication = "anonymous"
+VirtualHost "guest.meet.jitsi"
```

Comment:
This is not correct. It should be guest.jitsi.example.com.
It may work, but there are places in code where from a domain we remove to main virtualhost domain part if it exists and there it may fail.

Reply:
But the Docker setup uses exactly the same method. I wanted to keep almost the same config.

Do you think that I should update this part?

Reply:
> But the Docker setup uses exactly the same method. I wanted to keep almost the same config.

Nope, it doesn't mix domains. It uses meet.jitsi everywhere. What I say is that it should be consistent. Stick to the same main virtualhost the one that is configured as: https://github.com/jitsi/jitsi-meet/blob/d6776f234d5c847a7bfe0e483959be3ce79388ab/doc/debian/jitsi-meet-prosody/prosody.cfg.lua-jvb.example#L7